Section 3.md

```diff
@@ -140,6 +153,14 @@ Given its very limited production volume, Sterling features exorbitant profit ma
 
 Relatively underwhelming compared to medical giants such as Vey-Med, SaniCorp's products are largely behind-the-scenes medical gear. Many sanitation sprayers, gauze, sterile gloves, and autoinjector casings are made by SaniCorp. There is one notable exception to their somewhat pedestrian medical offerings, their prosthetics. While not offering as many flashy options as some augments, SaniCorp offers a range of comfortable, easy to maintain, clean, and operate prosthetic limb replacements and augmentations.
 
+- **Bishop Cybernetics**
+
+Bishop was founded in the earlier 24th century, just after the boom in robotics. Whilst Zeng-Hu Robotics was taking the field by storm by offering replacements that resembled real limbs, Bishop was in the background, making its money by selling sleek prosthetics, focusing on aesthetic and futurism, tapping into the market of transhumanists, something which was growing at a rapid pace during this time.
+
+Outside of their prosthetics, they also built entire chassis for people to inhabit, being some of the earliest adopters of the �Man Machine Interface� on a wide scale. These were remarked as the highest quality of a chassis you could get at the time, also being ludicrously expensive.
```

Comment:
"Outside of their prosthetics, Bishop also led the development on full body chassis development, and were one of the earliest adopters of Man Machine Interface (MMI, Pioneered by the Morpheus Corporation) on a wide scale. These were remarked as some of the highest quality of chassis available at the time, but were as well ludicrously expensive."